### PR TITLE
Clean up `Do you really want to ask peppy?` 

### PR DESCRIPTION
**Pull request by Sitoria**
Forum PM is end of service so just clean up this word

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Do_you_really_want_to_ask_peppy/en.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Do you really want to ask peppy?
 
-Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum PMs is quite hard to keep up with. He therefore asks that you do one of the following options:
+Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum and PMs are quite hard to keep up with. He therefore asks that you do one of the following options:
```
Comment (aceticke) [inline wiki/Do_you_really_want_to_ask_peppy/en.md:3]:
```suggestion
Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering the forum and DMs are quite hard to keep up with. He therefore asks that you do one of the following options:
```

File: wiki/Do_you_really_want_to_ask_peppy/en.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-## Technical Support
+## Technical Support <!-- i think need to insert: Tech Support in github but i dont know what repo to put in -->
```
Comment (aceticke) [inline wiki/Do_you_really_want_to_ask_peppy/en.md:5]:
```suggestion
## Technical Support
```

users should always attempt the help forum first. bugs relating to stable are few and far between, if the support team concurs its a genuine issue then they can direct them to https://github.com/ppy/osu-stable-issues/issues

File: wiki/Do_you_really_want_to_ask_peppy/en.md [hunk 3]
```diff
@@ -12,4 +12,4 @@ For account issues, problems with payments, ban follow-ups, enquiries about tour
 
 ## Contacting peppy
 
-If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh). Please be aware that any support or account-related requests to this email will be marked as spam.
+If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh) or Chat `#general` in [osu! Discord Server](/wiki/Community/osu!_Discord_server). Please be aware that any support or account-related requests to this email will be marked as spam.
```
Comment (aceticke) [inline wiki/Do_you_really_want_to_ask_peppy/en.md:15]:
```suggestion
If you wish to contact peppy personally for non-osu! related reasons, you can email him at [pe@ppy.sh](mailto:pe@ppy.sh). Please be aware that any support or account-related requests to this email will be marked as spam.
```

the osu discord is not a good way to get in contact with peppy personally, this isn't a good thing to encourage

**Review by aceticke — CHANGES_REQUESTED** [review]

File: wiki/Do_you_really_want_to_ask_peppy/en.md (hunk 2 above)
Comment (Sitoria) [reply wiki/Do_you_really_want_to_ask_peppy/en.md]: ok i talked this with Civil oath so deleted thx for review

File: wiki/Do_you_really_want_to_ask_peppy/en.md (hunk 1 above)
Comment (Sitoria) [reply wiki/Do_you_really_want_to_ask_peppy/en.md:3]: ah nice point thx

File: wiki/Do_you_really_want_to_ask_peppy/en.md (hunk 3 above)
Comment (Sitoria) [reply wiki/Do_you_really_want_to_ask_peppy/en.md]: okay i'll delete it sry for guess it

File: wiki/Do_you_really_want_to_ask_peppy/en.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Do you really want to ask peppy?
 
-Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum PMs is quite hard to keep up with. He therefore asks that you do one of the following options:
+Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering the forum and DMs are quite hard to keep up with. He therefore asks that you do one of the following options:
```
Comment (Walavouchey) [inline wiki/Do_you_really_want_to_ask_peppy/en.md:3]: this is the page that is opened when you try to dm peppy in-game. not sure what "answering forums" means but it's not relevant (how do you "contact" peppy via the forums?)

```suggestion
Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering DMs is quite hard to keep up with. He therefore asks that you do one of the following options:
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Do_you_really_want_to_ask_peppy/en.md (hunk 4 above)
Comment (Sitoria) [reply wiki/Do_you_really_want_to_ask_peppy/en.md:3]: ah i dont know this wiki page is use for automaic peppy DM answer 💦
nice point, i'll fix

**Review by Walavouchey — APPROVED** [review]

**Comment by Walavouchey** [comment]
force-pushed to combine 10+ commits

**Pull request merged**